pkg/proc/proc.go

```diff
@@ -147,6 +147,39 @@ func Continue(dbp Process) error {
 				}
 			}
 			return conditionErrors(threads)
+		case curbp.Active && curbp.Tracepoint:
+			if curbp.returnInfo != nil {
+				curthread.Common().returnValues = curbp.Breakpoint.returnInfo.Collect(curthread)
+				return conditionErrors(threads)
+			}
+			// We have hit a function entry breakpoint, set a
+			// breakpoint at the return address and load return
+			// values.
+			selg := dbp.SelectedGoroutine()
+			topframe, retframe, err := topframe(selg, curthread)
+			if err != nil {
+				return err
+			}
+
+			if topframe.Inlined {
+				if err := next(dbp, false, true); err != nil {
+					return err
+				}
+				return Continue(dbp)
```

Comment:
I think it would be better to just let Continue repeat its loop again, instead of calling it recursively. It's what every other case is doing.

Reply:
Agreed.